go.mod

```diff
@@ -0,0 +1,3 @@
+module Source
```

Comment:
what's that file doing here? :)

Reply:
When I added a go extension to VSCode for a bit of help it complained until I added one of these.

Reply:
I didn't think it would do any harm to submit and it may save someone some time setting up.

Reply:
OK